### PR TITLE
Add the list of events available for the webhooks

### PR DESCRIPTION
**Pull request by NotBad4U**
This PR add the list of events available for the option `--event` with the `clever-tools` webhooks command.

NOTE: I tried to format this with tables but the rendering was not pretty and hard to read. I jumped in a more straightforward format.

### CODE REVIEW

File: commons/clever-tools/notifications.md [hunk 1]
```diff
@@ -77,6 +77,63 @@ and there issue this command:
...
+
+##### Examples:
+
+Get a message in a Slack channel when the deployment for an application failed:
+
+```
+clever webhooks add "The application failed to deploy" https://hooks.slack.com/services/xxxxxx/ --format slack --event DEPLOYMENT_FAIL
+```
+
+Prevent your accounting service when credits are added:
```
Comment (BlackYoup) [inline commons/clever-tools/notifications.md]: s/Prevent/Notify/ ?

Comment (NotBad4U) [reply commons/clever-tools/notifications.md]: Why not

Comment (urcadox) [reply commons/clever-tools/notifications.md]: Is this a question or are you going to make the change? ("Prevent" is incorrect here, it's not a question of style)

Comment (NotBad4U) [reply commons/clever-tools/notifications.md]: Done

**Pull request merged**